go/support_infrastructure/domain_service/domain_service_test.go

```diff
@@ -364,3 +384,30 @@ func generateGuard(t *testing.T) *tao.Guard {
 	}
 	return &guard
 }
+
+// cleanUpTPMTao runs the finalizer for TPMTao early then unsets it so it
+// doesn't run later. Normal code will only create one instance of TPMTao, so
+// the finalizer will work correctly. But this test code creates multiple such
+// instances, so it needs to call the finalizer early.
+func cleanUpTPMTao(tt *tao.TPMTao) {
```

Comment:
I believe these are copy-pasted from another part of the code. If so, please refactor to expose these rather than copying them over.